diesel_cli/src/infer_schema_internals/pg.rs

```diff
@@ -33,12 +32,12 @@ pub fn determine_column_type(
     // Postgres doesn't coerce varchar[] to text[] so print out a message to inform
     // the user.
     if let (true, Some(tpe)) = (is_array, diesel_alias_without_postgres_coercion) {
-        writeln!(
-            &mut stderr(),
+        tracing::info!("Cannot coerce varchar[] into text[]");
+        eprintln!(
```

Comment:
Just curious under what conditions this returns either an `Ok` or an `Err`? I see the `?` in the deleted code, and tried reading up on https://doc.rust-lang.org/std/macro.eprintln.html but couldn't quite understand

Reply:
The old code was from some time before `eprintln!` existed. It basically emulated what `eprintln` does internally by writing directly to stderr, with the difference that it would bubble up any potential error from write. I think `eprintln` just panics in that case, but given that this likely means that stderr is not there it implies that something is really broken. At that point there is not much we could do anyway, so I don't think that matters.